composer/loggers/tensorboard_logger.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2022 MosaicML Composer authors
+# SPDX-License-Identifier: Apache-2.0
+
+"""Log to `Tensorboard <https://www.tensorflow.org/tensorboard/get_started#:~:text=TensorBoard%20is%20a%20tool%20for,dimensional%20space%2C%20and%20much%20more./>`_."""
+
+from composer.loggers.logger import LogLevel
+from composer.core.state import State
+from typing import Any, Dict, Optional
+from composer.utils import dist
+from torch.utils.tensorboard import SummaryWriter
+from composer.loggers.logger_destination import LoggerDestination
+
+__all__ = ['TensorboardLogger']
+
+class TensorboardLogger(LoggerDestination):
+    def __init__(self,
+                 log_dir: str=None,
+                 run_name: Optional[str]= None,
+                 rank_zero_only: bool = True):
+        
+        self.run_name = run_name if run_name is not None else ''
+        self.log_dir = log_dir
+        self.writer = SummaryWriter(log_dir=self.log_dir, filename_suffix=self.run_name)
+        self._enabled = (not rank_zero_only) or dist.get_global_rank() == 0
```

Comment:
`dist` will not have been enabled until later, so safer to do:
```
self.rank_zero_only = rank_zero_only
```

and then, down in `log_data`, do:
```
if (not rank_zero_only) or dist.get_global_rank() == 0:
```
```

Reply:
oh good catch! done here: 
172c29b